fluent-package/debian/fluent-package.lintian-overrides

```diff
@@ -31,3 +26,5 @@ fluent-package: binary-or-shlib-defines-rpath opt/fluent/lib/ruby/gems/*/gems/no
 fluent-package: embedded-library opt/fluent/lib/ruby/gems/*/extensions/x86_64-linux/*/nokogiri-*/nokogiri/nokogiri.so: libxml2
 fluent-package: embedded-library opt/fluent/lib/ruby/gems/*/gems/nokogiri-*/ext/nokogiri/nokogiri.so: libxml2
 fluent-package: embedded-library opt/fluent/lib/ruby/gems/*/gems/nokogiri-*/lib/nokogiri/nokogiri.so: libxml2
+fluent-package: unstripped-binary-or-object opt/fluent/lib/ruby/gems/3.2.0/extensions/x86_64-linux/3.2.0/debug-1.7.1/debug/debug.so
+fluent-package: unusual-interpreter opt/fluent/lib/ruby/gems/3.2.0/gems/ffi-1.15.5/ext/ffi_c/libffi/generate-darwin-source-and-headers.py #!python
```

Comment:
nits: Shouldn't we use `*` for version numbers?

Reply:
Thanks, as you mentioned, it should use `*` for it.